### PR TITLE
Add classes to scrap the edt website for the formations and the groups and register to local database

### PR DESCRIPTION
**Pull request by tbille**
Work in progress : do not merge

### CODE REVIEW

File: app/src/main/java/com/m2dl/mapus/mapus/model/Event.java [hunk 1]
```diff
@@ -0,0 +1,135 @@
+package com.m2dl.mapus.mapus.model;
+
+public class Event {
+
+    private int day;
+    private String prettyTime;
+    private String startTime;
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/model/Event.java:7]: Use another type (Date, Time ...) or also group (number of week/day of week/Time into one object)

File: app/src/main/java/com/m2dl/mapus/mapus/model/Event.java [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    private String endTime;
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/model/Event.java:8]: Use another type (Date, Time ...)

File: app/src/main/java/com/m2dl/mapus/mapus/edt/Timetable.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+                        event.select("starttime").text(),
+                        event.select("endtime").text(),
+                        event.select("category").text(),
+                        event.select("prettyweeks").text(),
+                        event.select("rawweeks").text(),
+                        event.select("module").text(),
+                        event.select("group").text(),
+                        event.select("room").text() ,
+                        event.select("notes").text()
+                ));
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/edt/Timetable.java]: Nothing is done for the moment with those values

File: app/src/main/java/com/m2dl/mapus/mapus/database/EventDatabase.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    public static final String COLUMN_PRETTY_WEEK = "prettyweek";
+    public static final String COLUMN_WEEK_NUMBER = "weeknumber";
+    public static final String COLUMN_MATIERE = "matiere";
+    public static final String COLUMN_GROUP = "groupe";
+    public static final String COLUMN_SALLE = "salle";
+    public static final String COLUMN_NOTES = "notes";
+
+    private static final String DATABASE_NAME = "events.db";
+    private static final int DATABASE_VERSION = 1;
+
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/database/EventDatabase.java]: remove line

Comment (tbille) [reply app/src/main/java/com/m2dl/mapus/mapus/database/EventDatabase.java]: done

File: app/src/main/java/com/m2dl/mapus/mapus/model/Event.java (hunk 2 above)
Comment (tbille) [reply app/src/main/java/com/m2dl/mapus/mapus/model/Event.java:8]: no for the moment we will use string

File: app/src/main/java/com/m2dl/mapus/mapus/model/Event.java (hunk 1 above)
Comment (tbille) [reply app/src/main/java/com/m2dl/mapus/mapus/model/Event.java:7]: no for the moment we will use string

**Review by ClementChaumel — APPROVED** [review]

**Pull request merged**